nginx-proxy/imgpath.js

```diff
@@ -1,5 +1,5 @@
 function file_path(r) {
-    var uri = decodeURI(r.uri);
+    var uri = decodeURI(r.uri).replace(/(https?):\/(\w)/i, '$1://$2');
```

Comment:
This is a whitelist approach, which would treat `http(s)://` explicitly, but could there be urls with double slashes in other cases (e.g. `ftp://` assuming thumbor supports it, or even just a mistake where there are two slashes?). If this happens then the signature is going to differ between nginx and thumbor

(this is another reason why I was hoping that the thumbor project would use something in the URL itself rather than a hash - it just makes those cases much easier... /cc @mpdude)

Reply:
I assumed Thumbor supports http&https only. I think the whitelist will work fine - the number of cases is limited.

Reply:
For now, the solution will work for (at least) most of the real world cases. There might be some other difference in url in nginx/python but it will not result in a huge problem - nginx will bypass these requests to thumbor.

Reply:
Ok, this seems like a pragmatic approach :+1:

Reply:
Ok, I've asked NSJ authors a question and they sent me a link http://nginx.org/en/docs/http/ngx_http_core_module.html#merge_slashes

I've tested it and it works just fine, there is no need in regex url patching with this.